server/migrations/20200805083200-create-validator-table.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, DataTypes) => {
+    return queryInterface.createTable('Validators', {
```

Comment:
we should add validator reward destination to this model so that we can check where rewards are issued. The options would be `stash-staked`, `stash-unstaked`, `controller`, and an arbitrary `AccountId` so they type should be a string.